.husky/commit-msg

```diff
@@ -1,4 +1,2 @@
-#!/usr/bin/env sh
```

Comment:
this was done automatically when I regenerated the lockfile. seems like the latest version of husky doesn't need it anymore